packages/documentation-v7/src/stories/components/blockquote/blockquote.docs.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Canvas, Controls } from '@storybook/blocks';
+import * as BlockquoteStories from './blockquote.stories.ts';
+
+<Meta of={BlockquoteStories} />
+
+# Blockquote
+
+<div className="lead">
+  For quoting blocks of content from another source within your document. Wrap `&lt;blockquote
+  class="blockquote"&gt;` around any HTML as the quote.
```

Comment:
This works not as I expected. Try this one and if this does also not work, implement the first solution without any ` around the blockquote.
```suggestion
  For quoting blocks of content from another source within your document. Wrap `<blockquote
  class="blockquote">` around any HTML as the quote.
```

Reply:
Sorry, I resolved the last one, but forgot to actually do it. It is fixed now.